dbt/adapters/clickhouse/connections.py

```diff
@@ -94,25 +96,24 @@ def open(cls, connection):
         kwargs = {}
 
         try:
-            handle = Client(
+            handle = clickhouse_connect.get_client(
                 host=credentials.host,
                 port=credentials.port,
                 database='default',
-                user=credentials.user,
+                username=credentials.user,
                 password=credentials.password,
-                client_name=f'dbt-{dbt_version}',
-                secure=credentials.secure,
-                verify=credentials.verify,
+                interface='https' if credentials.secure else 'http',
+                compress=False if credentials.compression == '' else bool(credentials.compression),
                 connect_timeout=credentials.connect_timeout,
                 send_receive_timeout=credentials.send_receive_timeout,
+                verify=True,
                 sync_request_timeout=credentials.sync_request_timeout,
```

Comment:
sync_request_timeout isn't a thing for http, you can accept the parameter but I wouldn't do anything with it.

Reply:
Removed it